xml-binding/2.3/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta XML Binding Specification 2.3"
+date: 2019-07-22T17:54:14+0200
+summary: "First release for Jakarta EE 8"
+---
+The Jakarta XML Binding provides an API and tools that automate the mapping
+between XML documents and Java objects.
+
+* [Jakarta XML Binding 2.3 Specification Document](./jakarta.xml.bind-spec-2.3.pdf) (PDF)
+* [Jakarta XML Binding 2.3 Specification Document](./jakarta.xml.bind-spec-2.3.html) (HTML)
+* [Jakarta XML Binding 2.3 Javadoc](./apidocs)
+* [Jakarta XML Binding 2.3 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/promoted/jaxb-tck-2.3_latest.zip)
```

Comment:
Did something change with our TCK publishing process?  This URL doesn't work.  Actually, all of the sub-directories under the top level directory have changed...  https://download.eclipse.org/ee4j/jakartaee-tck/.  This comment probably applies to all of the Java SE PRs.

Reply:
This is an (incorrect) reference to the staging directory.
The final TCK will be published under downloads.eclipse.org/jakarta/xml-binding/2.3/...

Reply:
done